roles/confluent.variables/vars/main.yml

```diff
@@ -350,7 +372,7 @@ kafka_connect_properties:
       producer.confluent.monitoring.interceptor.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | join(':' + kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']}}"
       consumer.confluent.monitoring.interceptor.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | join(':' + kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']}}"
   ssl:
-    enabled: "{{kafka_connect_ssl_enabled}}"
+    enabled: "{{kafka_connect_ssl_enabled or mds_broker_listener.ssl_enabled}}"
```

Comment:
mds only has 1 way tls so im guessing only a truststore is needed also the var is mds_ssl_enabled, connect does not connect to the listener